must-gather/must-gather-requirements.md

```diff
@@ -0,0 +1,60 @@
+# Requirements for OpenShift sandboxed containers must-gather
+
+`must-gather` for OpenShift sandboxed contains (OSC) should gather all information and logs needed for debugging OSC.
+
+### Usage
+```sh
+oc adm must-gather --image=registry.redhat.io/openshift-sandboxed-containers/osc-must-gather-rhel9:latest
+```
+
+The command above will create a local directory with a dump of the OpenShift sandboxed-containers state.
+Note that this command will only get data related to the sandboxed-containers part of the OpenShift cluster.
+
+### Contents of the dump
+- All namespaces (and their children objects) that belong to any sandboxed containers resources
+
+In order to get data about other parts of the cluster (not specific to sandboxed containers) you should
+run `oc adm must-gather` (without passing a custom image). Run `oc adm must-gather -h` to see more options.
+
+#### General
+- Resource definitions
+- Service logs
+
+#### OSC
+- All namespaces and their child objects that belong to any OpenShift sandboxed containers resources
+- All OpenShift sandboxed containers custom resource definitions (CRDs)
+  - sandboxed-containers/namespaces/openshift-sandboxed-containers-operator/**_*\_description_**
+
+- Kata agent logs **???????**
+- Kata runtime logs **??????**
```

Comment:
Part of the crio logs, as long as you have used the "loglevel=debug" flag in kataconfig.
You can find them by looking at the logs with "name=containerd-shim-v2" in them.

Alternatively, we can also get them separately using "journalctl -t kata" (still with the debug flag), but I don't think we're gathering that?

Reply:
Must-gather doesn't use journalctl anywhere